src/lib/components/templates/Chat.svelte

```diff
@@ -1,14 +1,43 @@
 <script>
 	import Header from './Header.svelte'
 	import type { ChatMessage } from '$types'
-	import { SpokenText, Help, Back } from '$atoms'
+	import { SpokenText, Help, Back, MessageCloud, AudioPlayer } from '$atoms'
 	import { AudioRecorder } from '$organisms'
 	import { client } from '$config/supabase'
+	import { onDestroy, onMount } from 'svelte'
+	import { resolveLetter } from '$db/letter'
+	import { postResolveStatusMessage } from '$db/messageHandler'
 
 	export let messages: ChatMessage[]
 	export let userRole: string
+	export let letterId: string
 
+	console.log(messages)
+
+	let form: HTMLFormElement
+	const medium: boolean = true
 	const userId = client.auth.session().user.id
+
+	$: radioVal = ''
+	$: isUser = userRole === 'user' ? true : false
+
+	if (userRole === 'user') {
+		onMount(async () => {
+			form.addEventListener('input', (e: Event) => {
```

Comment:
This can be done by putting `on:input` on the form tag in the template

Reply:
Even better, check out [`bind:group`](https://svelte.dev/docs#bind_group) on the svelte docs. This makes it possible to bind the `radioVal` variable to the currently checked input's value.